### DISCUSSION TITLE
Copy external static Markdown page pre-build

### DISCUSSION THREAD

**Discussion by yiliansource** [facebook/docusaurus#5680] [Q&A] [answered]
This might've already been asked, but I have not found anything thus far.

My project structure looks like this:

```
/docs
    /docs
        my-docs-page.md
    [...]
    docusaurus.config.js
/src
    [...]
README.md
CHANGELOG.md
```

The top-level `docs` folder contains the Docusaurus project for the website's documentation.

My problem is that I want to include the top-level `CHANGELOG.md` file as a static Markdown page at build-time, essentially treating it like it's just a regular Markdown file in `/docs/docs`. Is there a plugin that solves this, or is it possible via the lifecycle APIs?

**Comment by Josh-Cena** [comment, marked as answer]
The Docusaurus site already does exactly what you are looking for: https://github.com/facebook/docusaurus/blob/main/website/community/5-changelog.md

**Reply by yiliansource** [reply]
Ah, I didn't realize you could just import Markdown files like that, I was sure that would throw an exception at runtime. Thanks!

**Reply by slorber** [reply]
With MDX you should now be able to import any doc or React comp inside another (mdx files are converted to React components).

However there's currently a TOC limitation and you have to work around it a bit 🤪 https://github.com/facebook/docusaurus/issues/3915

**Reply by yiliansource** [reply]
@slorber That's completely fine, it's more than sufficient for me! 😝
